src/frontend/src/pages/ProjectDetailPage/ProjectForm/ProjectCreateContainer.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+import { useAllLinkTypes, useCreateSingleProject } from '../../../hooks/projects.hooks';
+import { mapBulletsToPayload } from '../../../utils/form';
+import { useToast } from '../../../hooks/toasts.hooks';
+import { CreateSingleProjectPayload } from '../../../utils/types';
+import { useState } from 'react';
+import ProjectFormContainer from './ProjectForm';
+import { ProjectFormInput } from './ProjectForm';
+import { useHistory } from 'react-router-dom';
+import { routes } from '../../../utils/routes';
+import { getRequiredLinkTypeNames } from '../../../utils/link.utils';
+import ErrorPage from '../../ErrorPage';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+
+const ProjectCreateContainer: React.FC = () => {
+  const toast = useToast();
+  const history = useHistory();
+
+  const name = String();
+  const budget = 0;
+  const summary = String();
+  const links: { linkId: string; url: string; linkTypeName: string }[] = [];
+  const goals: { bulletId: number; detail: string }[] = [];
+  const features: { bulletId: number; detail: string }[] = [];
+  const rules: { bulletId: number; detail: string }[] = [];
+  const constraints: { bulletId: number; detail: string }[] = [];
+  const teamId = '';
+  const [projectManagerId, setProjectManagerId] = useState<string | undefined>();
+  const [projectLeadId, setProjectLeadId] = useState<string | undefined>();
+  const crId = 0;
+  const carNumber = 0;
+
+  const { mutateAsync } = useCreateSingleProject();
```

Comment:
can u pull isLoading out of this as well and return  loading indicator if it's loading (same for edit)